dropwizard-core/src/main/java/io/dropwizard/server/AbstractServerFactory.java

```diff
@@ -489,14 +488,16 @@ protected Handler createAppServlet(Server server,
         handler.addFilter(ThreadNameFilter.class, "/*", EnumSet.of(DispatcherType.REQUEST));
         serverPush.addFilter(handler);
         if (jerseyContainer != null) {
-            String urlPattern = jerseyRootPath;
-            if (!urlPattern.endsWith("*") && !urlPattern.endsWith("/")) {
-                urlPattern += "/";
+            if (jerseyRootPath.isPresent()) {
+                String urlPattern = jerseyRootPath.get();
+                if (!urlPattern.endsWith("*") && !urlPattern.endsWith("/")) {
```

Comment:
:+1: just didn't want to have it in this initial commit - wanted to change as little as possible. should the move of the "normalizing" logic be it's own PR?